File: app/src/main/java/com/abrsoftware/myapplication/event/GeneralEvent.java
package com.abrsoftware.myapplication.event;

public class GeneralEvent {
    public static final int ERROR = 0;
    public static final int SUCCESS_RESPONCE = 1;
    public static final int SHOW_LOADING = 2;
    public static final int ERROR_RESPONCE = 4;

    private int typeEvent;
    private String message;
    private Object responce;

    public int getTypeEvent() {
        return typeEvent;
    }

    public void setTypeEvent(int typeEvent) {
        this.typeEvent = typeEvent;
    }

    public Object getResponce() {
        return responce;
    }

    public void setResponce(Object responce) {
        this.responce = responce;
    }
}

File: app/src/main/java/com/abrsoftware/myapplication/event/GeneralEvent.kt
package com.abrsoftware.myapplication.event

class GeneralEvent {
    var typeEvent = 0
    var responce: Any? = null

    companion object {
        const val ERROR = 0
        const val SUCCESS_RESPONCE = 1
        const val SHOW_LOADING = 2
        const val ERROR_RESPONCE = 4
    }
}